src/adventures-photography/_posts/2017-08-10-eurovelo-montenegro.md

```diff
@@ -0,0 +1,71 @@
+
+Our route along the valley floor undulated gently, taking us quickly to our days destination. On arrival in Sušanj we quickly located a restaurant to purchase some dinner where we could sit near the bikes, and using their WiFi followed our now normal city regime of booking a hotel on Booking.com while we waited for food to arrive. 
+
+Two family sized pizzas later we were rolling down to the sea-front before taking a slow ride along the path above the beach looking for the apartment. After a brief affair of struggling to find it in the back-streets we located two large gates and a man quickly approached us, opening the gates for us and pointing to where to leave the bikes and showing us to our tiny apartment — not much bigger than the two single-beds that occupied it. 
+
+{% figure %}
+  <div class="row pair">
+    {% img src: 2016-06-web-DSCF7477, alt: Skopje Bizzare. %}
+    {% img src: 2016-06-web-DSCF7479, alt: Flags, bridges and statues in Macedonia. %}
+  </div>
+
+  {% img src: 2016-06-web-DSCF7484, alt: Statue in Macedonia. %}
+
+  <div class="row pair">
+    {% img src: 2016-06-web-DSCF7481, alt: Statues of the famous in Skopje, Macedonia. %}
+    {% img src: 2016-06-web-DSCF7491, alt: Statues dwarfing multi-story hotels in Skopje, Macedonia. %}
+  </div>
+
+  {% figcaption %}Photoseries of Max during his repair, featuring a kitten of unknown origin that sought nothing but attention and someone to play with.{% endfigcaption %}
+{% endfigure %}
+
+We both took advantage of the early finish, and all the space in the secure courtyard to give the bikes a little affection before a long cool shower (having waited an hour for the boiler to warm up to get anything above freezing) and give some of our clothes a brief wash out. An early night would do us both good after the intensity of pushing through the remote regions of Albania. 
+
+---
+
+After a small lie-in we stood waiting for the electric gates to open at just gone 9:00. We quickly retraced out steps to the beach from the previous evening, briefly stopping at a shop to grab some breakfast and snacks, before sitting on the beach to eat. Both of the bikes were running much lighter than normal today, as we knew we’d be passing through towns and villages fairly regularly now meaning we didn’t need to worry about finding supplies until later in the day. 
+
+If all went to plan today we’d cross into Croatia before the day was out, putting us back on-schedule after the fiasco in Macedonia after finding our planned route was essentially impassable. 
```

Comment:
> after the fiasco in Macedonia after finding our planned route was essentially impassable

Is there something you can link to here?

Reply:
Good call, I'll do that and call name the correct country 😆